Add message processor and message store module.md files (#539)
wso2/ballerina-integrator: docs/content/src/learn/tutorials/messaging-integrations/reliable-delivery/src/message_processor/Module.md

Template for Message Processor Reliable Delivery

> # Service Orchestration 

Reviewer: Why are we calling this Service Orchestration?

Author: Fixed

This is a template for the [Reliable Delivery Tutorial](https://ei.docs.wso2.com/en/7.0.0/ballerina-integrator/learn/tutorials/messaging-integrations/reliable-delivery/1/). Please refer to it for more details on what you are going to build here. This template provides a starting point for your scenario. 

## Using the Template

> Run the following command to pull the `message_processor` and `message_store` template from Ballerina Central.

Reviewer: ```suggestion
Run the following commands to pull the `message_processor` and `message_store` template from Ballerina Central.
```

Author: Fixed

```
$ ballerina pull wso2/message_store
$ ballerina pull wso2/message_processor
```

Create a new project.

```bash
$ ballerina new reliable-delivery
```

> Now navigate into the above module directory you created and run the following command to apply the predefined template you pulled earlier.

Reviewer: ```suggestion
Now navigate into the above module directory you created and run the following commands to apply the predefined templates you pulled earlier.
```

Author: Fixed

```bash
$ ballerina add -t wso2/message_store message_store
$ ballerina add -t wso2/message_processor message_processor
```

This automatically creates reliable delivery service for you inside the `src` directory of your project.  

## Testing

First, start the message broker you installed previously. If you are using Active MQ, you can navigate to `<ACTIVE_MQ_HOME>/bin` 
and start the broker by running the command below.

```bash
$ ./activemq console
```

> Then let’s build the module. Navigate to the project root directory and execute the following command.

Reviewer: ```suggestion
Let’s build the module. Navigate to the project root directory and execute the following command.
```

Author: Fixed
